cmd/kava/cmd/iavlviewer/root.go

```diff
@@ -4,13 +4,16 @@ import (
 	"fmt"
 	"strconv"
 
+	"cosmossdk.io/log"
 	dbm "github.com/cometbft/cometbft-db"
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/server"
+	"github.com/cosmos/cosmos-sdk/store/wrapper"
 	ethermintserver "github.com/evmos/ethermint/server"
 	"github.com/spf13/cobra"
 
```

Comment:
house cleaning: can you remove newline and let fmt sort these?

Reply:
Do you mean remove line 6 or which one? 

We can use `goimports` for such type of thing if we need. But for common convention the grouping looks ok, as first are stdpackages, that goes first and separated.

Reply:
currently looks like:
```
import (
	"fmt"
	"strconv"

	"cosmossdk.io/log"
	dbm "github.com/cometbft/cometbft-db"
	"github.com/cosmos/cosmos-sdk/client"
	"github.com/cosmos/cosmos-sdk/server"
	"github.com/cosmos/cosmos-sdk/store/wrapper"
	ethermintserver "github.com/evmos/ethermint/server"
	"github.com/spf13/cobra"

	"github.com/cosmos/iavl"
	iavldb "github.com/cosmos/iavl/db"
)
```
removing line 14 will cause it to look like this:
```
import (
	"fmt"
	"strconv"

	"cosmossdk.io/log"
	dbm "github.com/cometbft/cometbft-db"
	"github.com/cosmos/cosmos-sdk/client"
	"github.com/cosmos/cosmos-sdk/server"
	"github.com/cosmos/cosmos-sdk/store/wrapper"
	"github.com/cosmos/iavl"
	iavldb "github.com/cosmos/iavl/db"
	ethermintserver "github.com/evmos/ethermint/server"
	"github.com/spf13/cobra"
)
```

Reply:
Ok, got it, you mean line 14, not 6